test/helpers/client_helper.rb

```diff
@@ -0,0 +1,35 @@
+##
+# Helper mixin for tests (especially compute-based ones). Provide access to the
+# client in use via the #client method to use.
+
+module ClientHelper
+  # Check to see if an operation is finished.
+  #
+  # @param op [Excon::Response] the operation object returned from an api call
+  # @return [Boolean] true if the operation is no longer executing, false
+  #   otherwise
+  def operation_finished?(op)
```

Comment:
We already have a pretty powerful set of helpers for tracking operations:
```
[13] pry(main)> operation = server.set_tags({"world"=>"1"})
=>   <Fog::Compute::Google::Operation
...
 status="PENDING",
...
[14] pry(main)> operation.wait_for{ ready? }
=> {:duration=>0}
[15] pry(main)> operation.status
=> "DONE"
```

So are you sure this is needed? I may be misinterpreting this though.